Representor.podspec

```diff
@@ -19,7 +19,7 @@ Pod::Spec.new do |spec|
     http_spec.subspec 'Core' do |core_spec|
       core_spec.dependency 'Representor/Core'
       core_spec.dependency 'Representor/Adapter/HAL'
-      core_spec.source_files = 'Representor/HTTP/*.swift', 'Representor/HTTP/Adapters/*.swift'
+      core_spec.source_files = 'Representor/HTTP/HTTP{Transition*,Deserialization}.swift', 'Representor/HTTP/Adapters/*.swift'
```

Comment:
Yeah, gonna restructure this in later pull request. Should be able to break it up more.